src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.wasm.cs

```diff
@@ -55,7 +55,7 @@ public static void SetCornerRadius(UIElement element, CornerRadius cornerRadius)
 			}
 			else
 			{
-				var borderRadiusCssString = $"{cornerRadius.TopLeft.ToStringInvariant()}px {cornerRadius.TopRight.ToStringInvariant()}px {cornerRadius.BottomRight.ToStringInvariant()}px {cornerRadius.BottomLeft.ToStringInvariant()}px";
+				var borderRadiusCssString = $"min(50%,{cornerRadius.TopLeft.ToStringInvariant()}px) min(50%,{cornerRadius.TopRight.ToStringInvariant()}px) min(50%,{cornerRadius.BottomRight.ToStringInvariant()}px) min(50%,{cornerRadius.BottomLeft.ToStringInvariant()}px)";
```

Comment:
Is there a way to make this string handing in TS instead? It should be less expensive.

Reply:
@carldebilly We are already calling `SetStyles` in this code path to set `overflow: hidden`. I guess if I'll be adding `min(50%, ...)` in JS this will be a separate interop call which is more expensive? Is there something I'm missing?